src/Oak/Cmd/Http/Options.js

```diff
@@ -0,0 +1,16 @@
+// DUPLICATED FROM VIRTUALDOM/NATIVE.JS
```

Comment:
Rather than duplicate, if this stuff is exactly the same, couldn't we make it reusable? Maybe a new module called something like Oak.Native.JSObject or something.

Reply:
I started working on this and it started pull on threads in the Oak.VirtualDom.Native module. I thought the ideal would be: